src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingBaker.java

```diff
@@ -316,4 +292,27 @@ public Window getWindow()
             return new WindowHutBaker(this);
         }
     }
+
+    public static class CraftingModule extends AbstractCraftingBuildingModule.Crafting
```

Comment:
I didn't want to put them in separate files since they're just policy modifications on the base class rather than "real" classes.  Different defaults rather than new functionality.  Having them as separate files would just increase the clutter in the directory tree.